x/babylon/keeper/handler_plugin.go

```diff
@@ -43,86 +40,35 @@ func NewCustomMsgHandler(k msKeeper, auth AuthSource) *CustomMsgHandler {
 	return &CustomMsgHandler{k: k, auth: auth}
 }
 
-// default authorization logic that ensures any max cap limit was set. It does not take the amount into account
-// as contracts with a limit 0 tokens may need to instant undelegate or run other operations.
-// Safety mechanisms for these operations need to be placed on the implementation side.
 func defaultMaxCapAuthorizator(k *Keeper) AuthSourceFn {
 	return func(ctx sdk.Context, contractAddr sdk.AccAddress) bool {
```

Comment:
Why don't you remove this? Just curious.

Reply:
This function is supposed to do access control for smart contracts, namely, enforcing that some messages can only be sent from certain contracts. This will be useful for us in the future, and I keep them as a reference for now. Added a TODO in the code to make this clear